Update keymanagement.md (#648)
usnistgov/FIPS201: _FIPS201/keymanagement.md

    [[SP 800-78]](../_Appendix/references.md#ref-SP-800-78).

The expiration date of the PIV authentication and card authentication certificates 

> **SHALL NOT** be after the expiration date of the PIV Card. The exiration of the PIV card is the is the date printed on the card (zone 14F). The CHUID contains the 'digital copy'/representation of that date. See section 4.2.1 for definition of the CHUID format. If the card is revoked, 

Reviewer: ```suggestion
**SHALL NOT** be after the expiration date of the PIV Card. 
The expiration of the PIV card is printed on the card in Zone 14F and is contained in the CHUID. See [section 4.2.1](#s-4-2-1) for definition of the CHUID format. 
If the card is revoked, 
```
